scripts/index.js

```diff
@@ -0,0 +1,107 @@
+const state = {
+    "temperature": 80,
+    "sky": "sunny",
+    "landscape": null,
+    "cityName": null,
+};
+
+const changeColorTemperatureRange = () => {
+    var temperatureDisplay = document.querySelector("#current-temp");
```

Comment:
remember to use `let` and `const`, not `var`. since the other variables you have declared used `const`, let's stick with that for consistency.

```suggestion
    const temperatureDisplay = document.querySelector("#current-temp");
```